Resources/views/Collector/statsd_prometheus.html.twig

```diff
@@ -19,9 +19,9 @@
 
 {% block menu %}
     <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon">{% apply spaceless %}
```

Comment:
>  the apply tag is available starting at twig >= 2.9

=> to avoid dropping older twig compatibility, please remove the `spaceless` thing and remove the extra spaces manually 🙏 

```suggestion
    <span class="icon"><img ... /></span>
```

FTR: [same PR for another repo](https://github.com/M6Web/AmqpBundle/pull/55)